features/cron.feature

```diff
@@ -123,7 +124,9 @@ Feature: Manage WP-Cron events and schedules
     When I run `wp cron event delete wp_cli_test_event_5`
     Then STDOUT should be:
       """
-      Success: Deleted 2 instances of the cron event 'wp_cli_test_event_5'.
+      Deleted the cron event 'wp_cli_test_event_5'
+      Deleted the cron event 'wp_cli_test_event_5'
+      Success: Deleted a total of 2 cron events.
```

Comment:
I'm double checking here.
Did the tests fail from the start? As I don't see you updated the scenario above here.
But it does seem correct.

Reply:
The scenario didn't change, the output did however. This is because both single and multiple cron hooks go through the same function. The 'deleted' counter now contains deletions from all hooks, instead of the events of a single hook.